custom_components/tuya_local/devices/carro_pn04f02d_fan_light.yaml

```diff
@@ -0,0 +1,45 @@
+name: Ceiling Fan and Light Switch
+product:
+  - id: 6ahfblexspuaem65
+    name: Carro PN-04F02D ceiling fan and light switch
+primary_entity:
+  entity: light
+  dps:
+    - id: 20
+      type: boolean
+      name: switch
+    - id: 21
+      type: string
+      # Shows in the instruction set with standard white/colour/scene/music properties
+      # but this is a dimming switch and doesn't have color/scene dps
+      name: work_mode
+      optional: true
+    - id: 22
+      name: brightness
+      type: integer
+      range:
+        min: 10
+        max: 1000
+      mapping:
+        - scale: 3.92
+    - id: 28
```

Comment:
Since this is not listed in status, I think this is never sent out by the device, and can be removed.  It is a write only value without adequate documentation, which can be used to change settings with transitions.